modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+If there is no other way to update a cluster, you can "force" an update by using the `--force` option in the CLI, which allows you to forcefully upgrade the cluster even when upgrade release image validation fails and the cluster is reporting errors.
+
+The following text will be returned by using the `$ oc adm upgrade --help` command in the CLI:
+
+`The cluster may report that the upgrade should not be performed due to a content verification error or update
+precondition failures such as operators blocking upgrades. Do not upgrade to images that are not appropriately signed
+without understanding the risks of upgrading your cluster to untrusted code. If you must override this protection use
+the --force flag.`
+
+It's important to understand the consequences of performing a forced update to determine if this decision outweighs the potential risks. 
+
+The Cluster Version Operator (CVO) runs precondition checks before starting an update to determine if the cluster has the `Upgradeable=false` status condition, or any other warning condition.
```

Comment:
Besides, CVO verifies the target release signature. If the verification fails, we will get `ReleaseAccepted=False` condition. e.g.

```
# oc adm upgrade 
warning: Cannot display available updates:
  Reason: VersionNotFound
  Message: Unable to retrieve available updates: currently reconciling cluster version 4.13.0-0.nightly-2023-10-14-180128 not found in the "stable-4.13" channel

Cluster version is 4.13.0-0.nightly-2023-10-14-180128

Upgradeable=False

  Reason: AdminAckRequired
  Message: Kubernetes 1.27 and therefore OpenShift 4.14 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958395 for details and instructions.

ReleaseAccepted=False

  Reason: RetrievePayload
  Message: Retrieving payload failed version="" image="registry.ci.openshift.org/ocp/release@sha256:3c5976a39479e11395334f1705dbd3b56580cd1dcbd514a34d9c796b0a0d9f8e" failure=The update cannot be verified: unable to verify sha256:3c5976a39479e11395334f1705dbd3b56580cd1dcbd514a34d9c796b0a0d9f8e against keyrings: verifier-public-key-redhat

Upstream is unset, so the cluster will use an appropriate default.
Channel: stable-4.13
```

Reply:
included a reference to the release signature verification :)